jenkins/databricks/build.sh

```diff
@@ -285,6 +286,12 @@ set_dep_jars()
     artifacts[DELTA]="-DgroupId=io.delta -DartifactId=delta-core_${SCALA_VERSION}"
     dep_jars[DELTA]=${dep_jars[SQL]}
 
+    # log4j-core
+    if [[ "$BASE_SPARK_VERSION" == "3.3.0" ]]; then
```

Comment:
Just a question on this- Since it is comparing to a specific Spark base version. Do we have to deal with it again during next Databricks release?

Reply:
yes, this is correct